azure-pipelines.yml

```diff
@@ -79,7 +79,8 @@ steps:
     AZURE_DOMAIN=$domain
```

Comment:
Minor, but it might be simpler to insert the file read into the here-string:

```powershell
$content = @"
AZURE_DOMAIN=$domain
$(Get-Content -Path '.env')
"
```

Then you won't need to change anything else...